src/js/components/Data/Data.js

```diff
@@ -28,7 +28,6 @@ export const Data = ({
   properties,
   toolbar,
   total,
-  updateOn = 'submit',
```

Comment:
So there is no `updateOn` anymore for Data we will need to update 
DataSearch/stories/simple.js
DataTableColumns/stories/simple.js

Reply:
Good call, done.